Base.rte/Devices/Tools/Constructor/Constructor.lua

```diff
@@ -82,21 +94,27 @@ function Create(self)
 
 	self.fireTimer = Timer();
 	self.fired = false;
+	
+	self.startresource = 3;	-- how many blocks of concrete to start with
 
 	self.buildTimer = Timer();
 	self.buildlist = {};
-	self.resource = 1;
-
+	self.resperbit = 80;	-- how much resource is required per one build bit
+							-- concrete collect/build ratio is 1:1
```

Comment:
what's up with these ugly comments?
Also, resperbit is a truly awful name. It's all lower case so the separator for each word in it isn't obvious, and it's quite meaningless even so. How about resourcesPerPixel or something? Assuming a bit is a pixel??

Reply:
Technically the "bit" here is 2x2 pixels, hence not naming it per pixel...
Also, a lot of these thousand-line scripts don't follow pascalCase so sometimes I leave it the way everything else is merely for coherency.

Reply:
Okay, I'll have to look through this more carefully to understand why the concept of a 2x2 pixel box is necessary. I guess it's for drawing to speed up the process, but it all seems a little arbitrary. And I hear you on naming, they should be fixed up sometime to match standards, but until they are, keeping consistency is a reasonable decision. I'll mark this resolved, but please do clean up the comments even if you keep the ugly name for now.